hw/ip/otbn/dv/otbnsim/README.md

```diff
@@ -0,0 +1,18 @@
+# OTBNSim - OTBN Simulation Software
+
+`OTBNSim` class represents the ISS in a traditional sense.
+It has methods for starting the executing, cycle-accurate simulation tracing and also abilty to simulate injected errors and wiping as a response to those errors.
+As explained in the overall [OTBN DV documentation](../../doc/dv/_index.md), this simulator is driven essentially from the SystemVerilog model of the OTBN module.
+
+Simulator works in a step-by-step fashion and also has multiple different methods to mimic external stimulus to OTBN.
+At a typical run without errors ISS does the following:
+ 1. Decode the program on the path given by `iss_wrapper.cc` by using `decode_file` method in `decode.py`/
+ 2. Loading decoded program to a local list in `sim.py`.
+ 3. With each `step` command from the SystemVerilog side, update simulated state of the core (`state.py`), registers (`wsr.py`, `csr.py` and `gpr.py`) and data memory (`dmem.py`).
+ 4. Once the step is done, pass the generated trace to `iss_wrapper.cc` to then get it passed to `OTBNTraceChecker`.
+
+## Co-Simulation with RTL
+Co-simulation works by using a tracer module (`otbn_tracer`) that spits out some logs from the RTL and having a serialised result in a text form on `stdout` to produce same style of logs from the Python model.
```

Comment:
```suggestion
For co-simulation of RTL and ISS, the `otbn_tracer` module logs state changes of the RTL, and the ISS logs state changes of the Python model.
```
I don't think the `stdout` mechanics should be in the first sentence explaining co-simulation, so I removed it. If you think it should be mentioned at this point, I suggest adding a second sentence.